modules/nf-core/nacho/normalise/environment.yml

```diff
@@ -0,0 +1,12 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
```

Comment:
I think you can remove this line

Reply:
This is present in the template:
https://github.com/nf-core/tools/blob/main/nf_core/module-template/environment.yml